app/src/main/java/com/kks/nimblesurveyjetpackcompose/util/DateUtil.kt

```diff
@@ -0,0 +1,14 @@
+package com.kks.nimblesurveyjetpackcompose.util
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+const val EEEE_MMM_D = "EEEE, MMM d"
+
+object DateUtil {
+
+    private val EEEE_comma_MMMMM_d =
+        SimpleDateFormat(EEEE_MMM_D, Locale.ENGLISH)
+
+    fun getBeautifiedCurrentDate(): String = EEEE_comma_MMMMM_d.format(Date())
```

Comment:
Let's keep things reusable 😉 
```suggestion
    fun getBeautifiedDate(date: Date = Date()): String = EEEE_comma_MMMMM_d.format(date)
```